charts/ups-broker/values.yaml

```diff
@@ -3,3 +3,5 @@
 image: quay.io/kubernetes-service-catalog/user-broker:v0.0.11
 # ImagePullPolicy; valid values are "IfNotPresent", "Never", and "Always"
 imagePullPolicy: Always
+# Specifies whether to deploy the Broker service-catalog object for this Deployment
+deployBrokerResource: true
```

Comment:
Can we change this to deployUPSBroker
I personally think it would be better to drop the Resource from the name, but the biggest thing missing is the fact that it's UPS broker specifically.